test/system/950-auth-scripts.bats

```diff
@@ -28,6 +28,6 @@ exit 43
 EOF
     chmod +x $auth_script
 
-    PODMAN_AUTH_SCRIPTS_DIR=$auth_dir run_podman 42 pull foobar
-    PODMAN_AUTH_SCRIPTS_DIR=$auth_dir run_podman 43 pull barfoo
+    PODMAN_PREEXEC_HOOKS_DIR=$auth_dir run_podman 42 pull foobar
+    PODMAN_PREEXEC_HOOKS_DIR=$auth_dir run_podman 43 pull barfoo
```

Comment:
Would you consider renaming the file, updating the header comments and test name, and renaming the `$auth_dir` variable? Otherwise future maintainers are going to be really lost.